### PR TITLE
Updated because codare.fun site is updated.

### PR DESCRIPTION
**Pull request by baybeku**
i updated this presence because https://codare.fun is updated.

6Lfu27Q

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy errors

File: websites/C/CodAre/presence.ts [hunk 1]
```diff
@@ -1,79 +1,115 @@
...
+        ).innerHTML +
+        ")";
+    }
+    if (document.location.pathname.toLowerCase().includes("/arama")) {
+      presenceData.details = "Bir kod arıyor:";
+      if (!searchCategory)
+        presenceData.state =
+          searchResult.charAt(0).toUpperCase() +
+          searchResult.slice(1).toLocaleString();
+      if (searchCategory)
```
Comment (Bas950) [inline websites/C/CodAre/presence.ts:47]: why not just use else?

File: websites/C/CodAre/presence.ts [hunk 2]
```diff
@@ -1,79 +1,115 @@
...
+    }
+    if (window.location.pathname.toLowerCase() === "/admin/kodlar") {
+      presenceData.details = "Bir admin sayfası görüntülüyor:";
+      presenceData.state = "Kodlar";
+    }
+    if (window.location.pathname.toLowerCase() === "/admin/yorumlar") {
+      presenceData.details = "Bir admin sayfası görüntülüyor:";
+      presenceData.state = "Yorumlar";
+    }
+    if (window.location.pathname.toLowerCase() === "/kategori/js") {
```
Comment (Bas950) [inline websites/C/CodAre/presence.ts:79]: ever heard of `else if`?

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]
Merry Christmas 🎉 and GJ

**Pull request merged**